src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -796,4 +802,39 @@ public Boolean getPresentationScoreEnabled() {
     public void setPresentationScoreEnabled(Boolean presentationScoreEnabled) {
         this.presentationScoreEnabled = presentationScoreEnabled;
     }
+
+    public List<GradingCriterion> getGradingCriteria() {
+        return gradingCriteria;
+    }
+
+    public Exercise gradingCriteria(List<GradingCriterion> gradingCriteria) {
+        this.gradingCriteria = gradingCriteria;
+        if (gradingCriteria != null) {
+            this.gradingCriteria.forEach(gradingCriterion -> {
```

Comment:
The code here and in line `833` are the same. Maybe you can move it to a private method? :)